cluster-scope/overlays/prod/emea/morty/secret-mgmt/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - openshift-config
+  - openshift-ingress
+  - openshift-monitoring
```

Comment:
is `opf-alertreceiver`  missing from this list intentional ?

Reply:
fixed